apps/omg_api/lib/state/core.ex

```diff
@@ -388,38 +386,38 @@ defmodule OMG.API.State.Core do
   def exit_utxos(exiting_utxos, %Core{utxos: utxos} = state) do
     exiting_utxos =
       exiting_utxos
-      |> Enum.filter(fn %{utxo_pos: utxo_pos} ->
-        Map.has_key?(utxos, Utxo.Position.decode(utxo_pos))
-      end)
+      |> Enum.filter(&utxo_exists?(&1, state))
 
     event_triggers =
       exiting_utxos
       |> Enum.map(fn %{owner: owner, utxo_pos: utxo_pos} ->
         %{exit: %{owner: owner, utxo_pos: Utxo.Position.decode(utxo_pos)}}
       end)
 
-    state =
+    new_utxos_in_state =
```

Comment:
That name is misleading. Those are not new utxos. Those are utxos that survived exits.